packages/material-ui/src/SvgIcon/SvgIcon.test.js

```diff
@@ -39,55 +38,70 @@ describe('<SvgIcon />', () => {
   );
 
   it('renders children by default', () => {
-    const wrapper = shallow(<SvgIcon>{path}</SvgIcon>);
-    expect(wrapper.contains(path)).to.equal(true);
-    expect(wrapper.props()['aria-hidden']).to.equal(true);
+    const { container, queryByTestId } = render(<SvgIcon>{path}</SvgIcon>);
+    const root = container.querySelector(`.${classes.root}`);
```

Comment:
```suggestion
    const root = container.firstChild;
```

*x* all the other assertions